swarm/output_processor.py

```diff
@@ -56,26 +58,27 @@ def get_results(self):
             thumbnail, video_buffer = make_video(
                 self.intermediate_images + [self.outputs[0].convert("RGB")], 5
             )
-            thumbnail_buffer = image_to_buffer(thumbnail, "image/jpeg", "web_low")
+            thumbnail_buffer = image_to_buffer(
+                thumbnail, "image/jpeg", "web_low")
             results["inference_video"] = make_result(
                 video_buffer, thumbnail_buffer, "video/webm"
             )
 
         return results
 
 
-def make_result(buffer, thumb, content_type):
+def make_result(_buffer, thumb, content_type):
```

Comment:
why _buffer?

Reply:
Changed due to usage in BytesIO although I have reviewed again and I don't think its a required change so its your choice. It was amended throughout the script.